### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for `overflow-*`

### ISSUE THREAD

**Issue by coderaiser** [stylelint/stylelint#8339]
### What is the problem you're trying to solve?

I have next code:

```css
@supports (overflow: overlay) {
    .files {
        overflow-y: overlay;
    }
}
```

As I see:

> Note: The keyword value `overlay` is a legacy value alias for `auto`. With overlay, the scroll bars are drawn on top of the content instead of taking up space.

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y

### What solution would you like to see?

Would be great to have auto fix for this:

```css
@supports (overflow: overlay) {
    .files {
        overflow-y: auto;
    }
}
```

---

`overlay` is not the only case.

e.g.
```css
a {
  overflow-x: -moz-hidden-unscrollable;
}
```

We just need to reuse [`<-non-standard-overflow>`](https://csstree.github.io/docs/syntax/#Type:-non-standard-overflow).

**Comment by Mouvedia** [comment]
Hello @coderaiser,

There are values not currently recognised by `css-tree`.
[`overlay`](https://csstree.github.io/docs/syntax/#Property:overflow-y) is one of these.

In the meantime you can use [`propertiesSyntax` option](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#propertiessyntax--property-syntax-) to extend `overflow-y`, `overflow-x` and `overflow`.

**Comment by jeddy3** [comment]
Thanks for the request and for using the template.

@Mouvedia am I right to think that https://github.com/stylelint/stylelint/pull/8340 will address the request to have `overflow-y: overlay;` auto fixed to `overflow-y: auto;`?

**Comment by Mouvedia** [comment]
> [@Mouvedia](https://github.com/Mouvedia) am I right to think that [#8340](https://github.com/stylelint/stylelint/pull/8340) will address the request to have `overflow-y: overlay;` auto fixed to `overflow-y: auto;`?

No it's not.
It just alerted me that something was wrong with `declaration-property-value-keyword-no-deprecated`.
i.e. you have changed his request from `declaration-property-value-no-unknown` to `declaration-property-value-keyword-no-deprecated`

---

The autofix that he is asking for, cannot be done on a `no-unknown` rule.
Ill change the title to the intent of the original poster.

**Comment by romainmenke** [comment]
Hi @coderaiser,

This was added upstream in: https://github.com/csstools/postcss-plugins/blob/main/packages/css-syntax-patches-for-csstree/CHANGELOG.md#1020

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AzAlgGxgZwATAB0A7AgFwgDcYAnFDCAdwFoAPeXSmjAQwE8BuYmU616zXuxE8BQ8lVGMmAI3pgA1pPnTBJOTTqK0RDEZiaufHQF8QAGhDosAOW4BbOIhgs3AByxIwbGw7cAgidABzBBAvHwhqUlxYFG4AVwxEwl1qdJx2LLJCkFgwHmpuUjQwph9qCB8aUl4mCm4MVJgmIggmVKI1boYSBFxSHJhZK1tiKx0QyHC0CIAxeNcK6OwmrBMiUgCwyKRXACtg+1gfYMQs3FwCEC3eHaNSB-YHnlIcN7tiO4eTxeeyYC0iTC23CIUG41Cg73uIC+PweMxAViAA)

Updating your dependencies will pull in the patch release of `@csstools/css-syntax-patches-for-csstree`.
